app/helpers/option_set_helper.rb

```diff
@@ -3,4 +3,8 @@ def degree_status_text(option)
     key = option.value.parameterize(separator: "_")
     I18n.t("helpers.answer.mailing_list_steps_degree_status.degree_status.#{key}", **{ default: option.value }.merge(Value.data))
   end
+
+  def format_option_value(option, translation_key = nil)
```

Comment:
I've added this helper to be general purpose for translating picklist options by the id rather than their current value. With a bit of refactoring we could remove the above degree status specific helper in favour of this and keep everything generic rather than step specific.